hbw/app/javascript/packs/hbw/components/form/select.js.jsx

```diff
@@ -200,16 +200,17 @@ modulejs.define('HBWFormSelect',
       fetchOptionsAsync = (inputValue, callback) => {
         const url = `${this.props.params.url}${(this.props.params.url.includes('?') ? '&' : '?')}`;
 
-        fetch(`${url}q=${inputValue}`)
-          .then(response => response.json())
-          .then((json) => {
-            const res = json.map(opt => ({
-              label: opt.text,
-              value: opt.id
-            }));
-
-            callback(res);
-          })
+        this.props.env.connection.request({
+          url:    `${url}q=${inputValue}`,
+          method: 'GET'
+        }).then((response) => {
```

Comment:
It looks fine, but we never used these methods with jQuery.ajax in this project. Did you checked this?

Reply:
I checked this, but I can change to `done` for identically

Reply:
No, it's OK if you checked this :)